scl/default-network-drivers/plugin.conf

```diff
@@ -24,6 +24,7 @@
 block source default-network-drivers(
 	udp-port(514)
 	tcp-port(514)
+	tcp-max-connections(10)
```

Comment:
Please rename it to `max-connections`, it's not only for tcp now.

Reply:
Done :)